### ISSUE TITLE
[REQUEST] Ground Control button feature

### ISSUE THREAD

**Issue by AirProxX** [flybywiresim/a32nx#2059]
**Is your feature request related to a problem? Please describe.**

**Describe the feature you'd like to see implemented**

**References**

**Additional context**

Discord username (if different from GitHub):hhep#3108 (pseudo on the group is AirProxX)

**Comment by hotshotp** [comment]
good idea, once the button is added i will make a sound for it as the cvr test button makes  a chirp sound

**Comment by AirProxX** [comment]
> good idea, once the button is added i will make a sound for it as the cvr test button makes a chirp sound

Hi mate, I'm new at the github group do you know why I'm not able to add any label on my post ? Thank you

**Comment by hotshotp** [comment]
I have implemented this in my latest PR
